File: JavaScript/js/exercicioManzandoPg46I.js
function media(){
    let contador = 1
    let soma = 0
    let media = 0
    while(contador <= 10){
        let valores = Number(prompt(`Digite o ${contador} número`))
        soma += valores
        media = soma / contador 
        contador++
    }
    document.getElementById('mostra').innerHTML = (`A soma dos valores digitados é ${soma} <br> A média aritimética dos valores é ${media}`)
}

File: Java/manzando/exercicioManzandoPg46I.java
package manzando;

import java.util.Scanner;

public class exercicioManzandoPg46I {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int contador = 1, soma = 0, media = 0;
		
		while(contador <= 10) {
			System.out.printf("Digite o %d° número\n", contador);
			int valores = sc.nextInt();
			soma += valores;
			media = soma / contador ;
			contador++;
		}
		
		System.out.printf("A soma dos valores digitados é %d \nA média aritimética dos valores é %d",soma, media);
		
		sc.close();
	}
}
